src/test/java/kitchenpos/order/service/OrderServiceTest.java

```diff
@@ -0,0 +1,268 @@
+package kitchenpos.order.service;
+
+import kitchenpos.application.OrderService;
+import kitchenpos.domain.*;
+import kitchenpos.infra.KitchenridersClient;
+import kitchenpos.menu.fixture.MenuFixture;
+import kitchenpos.order.fixture.OrderFixture;
+import kitchenpos.order.fixture.OrderLineItemFixture;
+import kitchenpos.order.fixture.OrderTableFixture;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.AdditionalAnswers;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+import java.util.Optional;
+
+@DisplayName("주문 서비스 테스트")
+@ExtendWith(MockitoExtension.class)
+public class OrderServiceTest {
+    @Mock
+    private OrderRepository orderRepository;
+    @Mock
+    private MenuRepository menuRepository;
+    @Mock
+    private OrderTableRepository orderTableRepository;
+    @Mock
+    private KitchenridersClient kitchenridersClient;
+    @InjectMocks
+    private OrderService orderService;
+
+    private OrderFixture orderFixture;
+    private OrderLineItemFixture orderLineItemFixture;
+    private OrderTableFixture orderTableFixture;
+    private MenuFixture menuFixture;
+
+    @BeforeEach
+    void setUp() {
+        orderFixture = new OrderFixture();
+        orderLineItemFixture = new OrderLineItemFixture();
+        orderTableFixture = new OrderTableFixture();
+        menuFixture = new MenuFixture();
+    }
+
+    @Test
+    @DisplayName("주문 시 주문 유형 및 항목은 반드시 존재 해야한다.")
+    void create_exception_status_item_null() {
+        List<Order> exceptionOrders = List.of(orderFixture.주문_유형_없는_주문, orderFixture.주문_항목_없는_주문);
+
+        for (Order exceptionOrder : exceptionOrders) {
+            Assertions.assertThatThrownBy(
+                    () -> orderService.create(exceptionOrder)
+            ).isInstanceOf(IllegalArgumentException.class);
+        }
+    }
+
+    @Test
+    @DisplayName("손님은 식당에서 음식을 주문할 수 있다.")
+    void create_eatIn() {
+        Order 매장_주문 = orderFixture.매장_주문_A;
+        OrderTable 주문_테이블 = orderTableFixture.손님_있는_주문_테이블;
+
+        mockingMenuRepository(RepositoryMethod.FIND, menuFixture.메뉴_A);
+        mockingMenuRepository(RepositoryMethod.FIND_ALL, menuFixture.메뉴_A);
+        mockingOrderTableRepository(주문_테이블);
+        mockingOrderRepository(RepositoryMethod.SAVE, 매장_주문);
+
+        Order result = orderService.create(매장_주문);
+        Assertions.assertThat(result.getOrderTable()).isEqualTo(주문_테이블);
```

Comment:
```suggestion
        Order result = orderService.create(매장_주문);
        
        Assertions.assertThat(result.getOrderTable()).isEqualTo(주문_테이블);
```
공백 라인을 활용하여 문단을 나누어 가독성을 높혀보세요 😄